.github/workflows/pr-labeler.yml

```
@@ -17,6 +17,7 @@ jobs:
           repoToken: "${{ secrets.GITHUB_TOKEN }}"
           commentOnDirty: "This pull request has conflicts, please resolve those before we can evaluate the pull request."
       - name: Apply Size Label
+        if: ${{ ! github.ref == 'refs/heads/master' }}
```

I'm unsure if the `!` applies before or after the `==`. Probably best to use `!=` instead.
```suggestion
        if: ${{ github.ref != 'refs/heads/master' }}
```